crates/uv/tests/lock_scenarios.rs

```diff
@@ -582,7 +582,7 @@ fn fork_filter_sibling_dependencies() -> Result<()> {
         version = "1.0.0"
         source = { registry = "https://astral-sh.github.io/packse/PACKSE_VERSION/simple-html/" }
         dependencies = [
-            { name = "package-d", version = "2.0.0", source = { registry = "https://astral-sh.github.io/packse/PACKSE_VERSION/simple-html/" } },
+            { name = "package-d", version = "2.0.0", source = { registry = "https://astral-sh.github.io/packse/PACKSE_VERSION/simple-html/" }, marker = "sys_platform == 'darwin'" },
```

Comment:
I think this one and the one above are superfluous (again, assuming a DAG traversal).